git/09-pullrequests.md

```diff
@@ -326,9 +326,7 @@ Web browser to your repository:
 
 > https://gitlab.cern.ch/YOUR_CERN_USERNAME/starter-kit-example
 
-Then click on the **+ (plus)** button and select **New merge request**:
-
-![New merge request](fig/gitlab-pr-newmergerequest.png)
+Then click on the **Merge Requests** button on the left hand side of the screen and select **New merge request**:
```

Comment:
Are you sure you want to remove the screenshot here? If yes, you might be able to remove the image file as well (as long as it's not used elsewhere).

Reply:
I will replace it with a newer screenshot